### PR TITLE
Tier5 step1

### PR DESCRIPTION
**Pull request by LinuxDonald**
Added very basic level up to tier5.

### CODE REVIEW

**Comment by LinuxDonald** [comment]
@janexx the building seems to big? is it 2x2?

**Comment by janexx** [comment]
@LinuxDonald yes you are right... Sorry ;) I upload the images agein...

**Comment by LinuxDonald** [comment]
@janexx have you time this weekend for rerender the building?

**Comment by janexx** [comment]
yes, I think so

**Comment by LinuxDonald** [comment]
Good Work @janexx :)

**Comment by janexx** [comment]
@LinuxDonald look with increased light intensity...
testrender
Looks better, doesn't it?

**Comment by LinuxDonald** [comment]
@janexx WoW NICE :+1:

**Comment by LinuxDonald** [comment]
@squiddy please review. And are the commit messanges okay?

**Comment by LinuxDonald** [comment]
[autosave-2017-06-04--18-23-10.sqlite.zip](https://github.com/unknown-horizons/unknown-horizons/files/1066141/autosave-2017-06-04--18-23-10.sqlite.zip)
Savegame for testing.

File: horizons/world/playerstats.py [hunk 1]
```diff
@@ -117,12 +117,14 @@ def _collect_info(self):
 			TIER.PIONEERS: 3,
 			TIER.SETTLERS: 7,
 			TIER.CITIZENS: 15,
+			TIER.MERCHANTS: 20, # FIXME: dont know what valve must be here
```
Comment (squiddy) [inline horizons/world/playerstats.py]: These were added in c6be172105a112399bddb3b10d3c6e63277932f1, the commit message suggests that they need proper balancing.

I'd say the values look fine, you can remove the FIXME.

**Comment by LinuxDonald** [comment]
I dont know why the test fail. Local i dont have any problems with saving/loading of tier5.

**Comment by LinuxDonald** [comment]
@AndyMender could you please check this? https://travis-ci.org/unknown-horizons/unknown-horizons/builds/249092192#L1136

**Comment by LinuxDonald** [comment]
@squiddy maybe you can take an look too?

Merged it to fast press go on wrong pr...

**Comment by AndyMender** [comment]
@LinuxDonald I'm looking at it right now. It seems to me that the level numbers are wrong for some reason. I'll test this locally to see what's going on.

**Pull request merged**